server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -506,14 +506,24 @@ public void reportShuffleResult(
         "appId[" + appId + "], shuffleId[" + shuffleId + "], taskAttemptId[" + taskAttemptId + "]";
 
     try {
+      int expected =
+          partitionToBlockIds.values().stream().map(x -> x.length).reduce(0, (a, b) -> a + b);
```

Comment:
This is more readable:
`int expected = partitionToBlockIds.values().stream().mapToInt(x -> x.length).sum();`

Reply:
Sounds good.